extensions/mssql/src/objectManagement/interfaces.ts

```diff
@@ -472,6 +473,16 @@ export interface DatabaseViewInfo extends ObjectManagement.ObjectViewInfo<Databa
 	azureMaxSizes?: AzureEditionDetails[];
 	pageVerifyOptions?: string[];
 	restrictAccessOptions?: string[];
+	dscOnOffOptions: string[];
```

Comment:
Should these be nullable fields for cases where we use the view info outside of the configurations tab?